### PR TITLE
Add Opensprinkler integration documentation

### PR DESCRIPTION
**Pull request by vinteo**
## Proposed change

Add Opensprinkler integration (new) documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/34146
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1330
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/opensprinkler.markdown [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  devices:
+    - host: YOUR_DEVICE_HOST_NAME
+      port: 8080
+      password: YOUR_MD5_PASSWORD
+      name: Backyard Sprinklers
+      run_seconds: 60
+```
+
+{% configuration %}
+devices:
```
Comment (klaasnicolaas) [inline source/_integrations/opensprinkler.markdown:41]: I'm missing the `run_seconds` option.

File: source/_integrations/opensprinkler.markdown [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    host:
+      description: The host name of your Opensprinkler device
+      required: true
+      type: string
+    password:
+      description: The **MD5'ed** version of your password.
+      required: true
+      type: string
+    port:
+      description: The port to the Opensprinkler device API. Defaults to 8080 which is the Opensprinkler default.
```
Comment (klaasnicolaas) [inline source/_integrations/opensprinkler.markdown]:
```suggestion
      description: The port to the Opensprinkler device API.
```

File: source/_integrations/opensprinkler.markdown [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      required: false
+      type: integer
+      default: 8080
+    name:
+      description: The name of your Opensprinkler. Defaults to `Opensprinkler`
+      required: false
```
Comment (klaasnicolaas) [inline source/_integrations/opensprinkler.markdown:61]: Add: `default: Opensprinkler`

File: source/_integrations/opensprinkler.markdown [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+      description: The **MD5'ed** version of your password.
+      required: true
+      type: string
+    port:
+      description: The port to the Opensprinkler device API. Defaults to 8080 which is the Opensprinkler default.
+      required: false
+      type: integer
+      default: 8080
+    name:
+      description: The name of your Opensprinkler. Defaults to `Opensprinkler`
```
Comment (klaasnicolaas) [inline source/_integrations/opensprinkler.markdown]:
```suggestion
      description: The name of your Opensprinkler.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/opensprinkler.markdown [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+## Configuration
+
+You will need the host name of your Opensprinkler controller and a **MD5'ed** version of your password.
+
+Add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+opensprinkler:
+  devices:
```
Comment (klaasnicolaas) [inline source/_integrations/opensprinkler.markdown:35]: Only required config options in the first example

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**